lib/proxy/helpers.rb

```diff
@@ -25,7 +25,8 @@ def log_halt code=nil, exception=nil
 
   # read the HTTPS client certificate from the environment and extract its CN
   def https_cert_cn
-    certificate_raw = request.env['SSL_CLIENT_CERT'].to_s
+    ssl_client_cert_header = Proxy::SETTINGS.ssl_client_cert_header || 'SSL_CLIENT_CERT'
+    certificate_raw = request.env[ssl_client_cert_header].to_s.gsub(/   /, "\n")
```

Comment:
RFC specifies single white space or a tab for header continuation, a '+' in ```split(/[ |\t]+/)``` isn't needed. I need to think over if we want to handle cases where header continuation is invalid, but I'm leaning towards "no". 

No need for the ```cert``` line before the end,  the result of the expression on the previous line is the same. I wouldn't build a string by reassigning it to itself -- I haven't looked, but I'd bet this incurs additional copying. Something like ```begin + ssl_client_cert.split(/[ |\t]+/)[2..-3].join("\n") + end ``` is probably preferable.

Lastly, could you add a test for this? ```sinatra/ssl_client_verification_test.rb``` looks like a good spot.